src/components/__tests__/rating.js

```diff
@@ -30,14 +34,16 @@ test('clicking a star calls updateListItem', () => {
   jest.useFakeTimers()
   fireEvent.click(oneStar)
   expect(oneStar.checked).toBe(true)
-  jest.runAllTimers()
+  act(() => jest.runAllTimers())
   jest.useRealTimers()
+
   expect(updateListItem).toHaveBeenCalledTimes(1)
   expect(updateListItem).toHaveBeenCalledWith(
     __mockListItemDispatch,
     listItem.id,
     {rating: 1},
   )
+  await act(() => fakeResponse)
```

Comment:
I only need to bother with this because there are no observable DOM changes (thanks to optimistic updates) even though there are some state updates. If there were, then I could use RTL's utilities which use async act under the hood.